lib/rollbar/request_data_extractor.rb

```diff
@@ -130,6 +136,21 @@ def rollbar_raw_body_params(rack_req)
       rack_req.body.rewind
     end
 
+    def json_request?(rack_req)
+      rack_req.env['CONTENT_TYPE'] =~ %r{application/json} ||
+      rack_req.env['ACCEPT']       =~ /\bjson\b/
+    end
+
+    def json_request_missing_content_type?(env)
```

Comment:
This is a little clumsier than I would like–but the added work on L1456-147 is necessary to abstract this method.